src/styles/blocks/header.scss

```diff
@@ -0,0 +1,29 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+
+  padding: 0 50px;
+  height: 100%;
+  max-width: 1024px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  @media (min-width: 1024px) {
+    min-width: 1024px;
+  }
+
+  @media (min-width: 1200px) {
+    min-width: 1200px;
+  }
```

Comment:
are you this one works properly? do you need it?

Reply:
you are absolutely right, I don't need it (